internal/eval/evalers.go

```diff
@@ -1139,9 +1207,13 @@ func newExtensionEval(name types.Path, args []Evaler) Evaler {
 
 	if i, ok := extensions.ExtMap[name]; ok {
 		if i.Args != len(args) {
-			return newErrorEval(fmt.Errorf("%w: %s takes %d parameter(s)", errArity, name, i.Args))
+			return newErrorEval(fmt.Errorf("%w: %s takes %d parameter(s), but %d provided", errArity, name, i.Args, len(args)))
 		}
 		switch {
+		case name == "hasTag":
+			return newHasTagEval(args[0], args[1])
+		case name == "getTag":
+			return newGetTagEval(args[0], args[1])
```

Comment:
`hasTag` and `getTag` are not extension methods. I would think they should be implemented as first class operators, like `contains()` and its ilk.

Reply:
Yes, done here: https://github.com/cedar-policy/cedar-go/pull/77/commits/4f433e34c428f2de144c83035ba1fd0a313a904c.